### ISSUE TITLE
1.0.1 sphinxcontrib-jsmath: pytest warning

### ISSUE THREAD

**Issue by kloczek** [sphinx-doc/sphinx#9361]
I'm not 100% sure is it right place to report pytest warnings in sphinxcontrib-jsmath but documentation of that module points to here
```
**+ PYTHONPATH=/home/tkloczko/rpmbuild/BUILDROOT/python-sphinxcontrib-jsmath-1.0.1-13.fc35.x86_64/usr/lib64/python3.8/site-packages:/home/tkloczko/rpmbuild/BUILDROOT/python-sphinxcontrib-jsmath-1.0.1-13.fc35.x86_64/usr/lib/python3.8/site-packages
+ PYTHONDONTWRITEBYTECODE=1
+ /usr/bin/pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.9, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
benchmark: 3.4.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /home/tkloczko/rpmbuild/BUILD/sphinxcontrib-jsmath-1.0.1
plugins: forked-1.3.0, shutil-1.7.0, virtualenv-1.7.0, expect-1.1.0, httpbin-1.0.0, flake8-1.0.7, timeout-1.4.2, betamax-0.8.1, freezegun-0.4.2, case-1.5.3, isort-1.3.0, aspectlib-1.5.2, asyncio-0.15.1, toolbox-0.5, xprocess-0.17.1, aiohttp-0.3.0, checkdocs-2.7.0, mock-3.6.1, rerunfailures-9.1.1, requests-mock-1.9.3, cov-2.12.1, pyfakefs-4.5.0, cases-3.6.1, flaky-3.7.0, hypothesis-6.14.0, benchmark-3.4.1, xdist-2.3.0, Faker-8.8.1
collected 5 items

. .                                                                                                                                                                  [ 25%]
tests/test_jsmath.py ...                                                                                                                                             [100%]

============================================================================= warnings summary =============================================================================
tests/test_jsmath.py::test_basic
  /home/tkloczko/rpmbuild/BUILD/sphinxcontrib-jsmath-1.0.1/tests/test_jsmath.py:17: RemovedInSphinx50Warning: Path.text() is deprecated.  Please use read_text() instead.
    content = (app.outdir / 'math.html').text()

tests/test_jsmath.py::test_numfig_enabled
  /home/tkloczko/rpmbuild/BUILD/sphinxcontrib-jsmath-1.0.1/tests/test_jsmath.py:37: RemovedInSphinx50Warning: Path.text() is deprecated.  Please use read_text() instead.
    content = (app.outdir / 'math.html').text()

tests/test_jsmath.py::test_disabled_when_equations_not_found
  /home/tkloczko/rpmbuild/BUILD/sphinxcontrib-jsmath-1.0.1/tests/test_jsmath.py:55: RemovedInSphinx50Warning: Path.text() is deprecated.  Please use read_text() instead.
    content = (app.outdir / 'index.html').text()

-- Docs: https://docs.pytest.org/en/stable/warnings.html
====================================================================== 4 passed, 3 warnings in 7.94s =======================================================================
```

**Comment by tk0miya** [comment]
Thank you for reporting.
Fixed by https://github.com/sphinx-doc/sphinxcontrib-jsmath/pull/10

**Comment by kloczek** [comment]
Just tested that RP
```console
[tkloczko@barrel SPECS]$ rpmbuild -ba --with check python-sphinxcontrib-jsmath.spec
warning: Downloading https://github.com/sphinx-doc/sphinxcontrib-jsmath//archive/1.0.1/python-sphinxcontrib-jsmath-1.0.1.tar.gz to /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-1.0.1.tar.gz
warning: Downloading https://github.com/sphinx-doc/sphinxcontrib-jsmath//commit/f7e35479.patch#/python-sphinxcontrib-jsmath-Fix-mypy-0.730-violations.patch to /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-Fix-mypy-0.730-violations.patch
warning: Downloading https://github.com/sphinx-doc/sphinxcontrib-jsmath//commit/8a765831.patch#/python-sphinxcontrib-jsmath-test-Do-test-with-py39.patch to /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-test-Do-test-with-py39.patch
warning: Downloading https://github.com/sphinx-doc/sphinxcontrib-jsmath//commit/631ef5fb.patch#/python-sphinxcontrib-jsmath-Fix-mypy-violations.patch to /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-Fix-mypy-violations.patch
warning: Downloading https://github.com/sphinx-doc/sphinxcontrib-jsmath//commit/3297b271.patch#/python-sphinxcontrib-jsmath-Fix-9361-RemovedInSphinx50Warning-on-testing.patch to /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-Fix-9361-RemovedInSphinx50Warning-on-testing.patch
warning: Downloading https://github.com/sphinx-doc/sphinxcontrib-jsmath//commit/5d6d5f34.patch#/python-sphinxcontrib-jsmath-CI-Install-types-setuptools-to-invoke-mypy.patch to /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-CI-Install-types-setuptools-to-invoke-mypy.patch
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.lTtyGt
+ umask 022
+ cd /home/tkloczko/rpmbuild/BUILD
+ cd /home/tkloczko/rpmbuild/BUILD
+ rm -rf sphinxcontrib-jsmath-1.0.1
+ /usr/bin/gzip -dc /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-1.0.1.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd sphinxcontrib-jsmath-1.0.1
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ /usr/bin/cat /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-Fix-mypy-0.730-violations.patch
+ /usr/bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch
+ /usr/bin/cat /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-test-Do-test-with-py39.patch
+ /usr/bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch
+ /usr/bin/cat /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-Fix-mypy-violations.patch
+ /usr/bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch
+ /usr/bin/cat /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-Fix-9361-RemovedInSphinx50Warning-on-testing.patch
+ /usr/bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch
+ /usr/bin/cat /home/tkloczko/rpmbuild/SOURCES/python-sphinxcontrib-jsmath-CI-Install-types-setuptools-to-invoke-mypy.patch
+ /usr/bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch
+ RPM_EC=0

[..]

Executing(%install): /bin/sh -e /var/tmp/rpm-tmp.9OkIdB

[..]

+ /usr/bin/python3 setup.py install -O1 --skip-build --root /home/tkloczko/rpmbuild/BUILDROOT/python-sphinxcontrib-jsmath-1.0.1-13.1.fc35.x86_64
running install
running install_lib

[..]

+ PYTHONPATH=/home/tkloczko/rpmbuild/BUILDROOT/python-sphinxcontrib-jsmath-1.0.1-13.1.fc35.x86_64/usr/lib64/python3.8/site-packages:/home/tkloczko/rpmbuild/BUILDROOT/python-sphinxcontrib-jsmath-1.0.1-13.1.fc35.x86_64/usr/lib/python3.8/site-packages
+ /usr/bin/pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.11, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
benchmark: 3.4.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /home/tkloczko/rpmbuild/BUILD/sphinxcontrib-jsmath-1.0.1
plugins: forked-1.3.0, shutil-1.7.0, virtualenv-1.7.0, expect-1.1.0, httpbin-1.0.0, flake8-1.0.7, timeout-1.4.2, betamax-0.8.1, freezegun-0.4.2, case-1.5.3, isort-1.3.0, aspectlib-1.5.2, asyncio-0.15.1, toolbox-0.5, xprocess-0.17.1, aiohttp-0.3.0, checkdocs-2.7.0, mock-3.6.1, rerunfailures-9.1.1, requests-mock-1.9.3, cov-2.12.1, pyfakefs-4.5.0, flaky-3.7.0, hypothesis-6.14.0, benchmark-3.4.1, xdist-2.3.0, pylama-7.7.1, datadir-1.3.1, regressions-2.2.0, cases-3.6.3, Faker-8.10.0
collected 5 items

. .                                                                                                                                                                  [ 25%]
tests/test_jsmath.py ...                                                                                                                                             [100%]

============================================================================ 4 passed in 7.50s =============================================================================
```
So all looks clean now.

Thak you :)
